.schema/config.schema.json

```diff
@@ -723,6 +723,13 @@
               "examples": [
                 true
               ]
+            },
+            "enforced_public_clients": {
```

Comment:
Nice, I think if we name it `enforced_for_public_clients` it's a bit easier to read :) Once that's done this is ready to be merged as far as I am concerned! :)

Reply:
fixed:+1: